src/Eurofurence.App.Server.Services/ArtistsAlley/TableRegistrationService.cs

```diff
@@ -97,7 +126,7 @@ public async Task RegisterTableAsync(ClaimsPrincipal user, TableRegistrationRequ
                 DisplayName = _htmlSanitizer.Sanitize(request.DisplayName),
                 WebsiteUrl = request.WebsiteUrl,
                 ShortDescription = _htmlSanitizer.Sanitize(request.ShortDescription),
-                TelegramHandle = request.TelegramHandle,
+                TelegramHandle = request.TelegramHandle.TrimStart('@'),
```

Comment:
Sending a request with no telegram handle will result in a 500 err.
A null check is probably needed here
```suggestion
                TelegramHandle = request.TelegramHandle?.TrimStart('@'),
```